docs/cluster-agent/CUSTOM_METRICS_PROVIDER.md

```diff
@@ -177,35 +191,60 @@ Voilà.
 
 - Make sure you have the Aggregation layer and the certificates set up as per the requirements section.
 - Always make sure the metrics you want to autoscale on are available.
-As you create the HPA, the DCA parses the manifest and queries Datadog to try to fetch the metric.
+As you create the HPA, the Datadog Cluster Agent parses the manifest and queries Datadog to try to fetch the metric.
 If there is a typographic issue with your metric name or if the metric does not exist within your Datadog application the following error is raised:
 ```
 2018-07-03 13:47:56 UTC | ERROR | (datadogexternal.go:45 in queryDatadogExternal) | Returned series slice empty
 ```
-You can check in the ConfigMap used to store and share the HPA state by the DCA:
-`kubectl get cm datadog-hpa -o yaml`
-yields:
+
+You can run the `datadog-cluster-agent status` command to see the status of the External Metrics Provider process:
+
+```
+  Custom Metrics Provider
+  =======================
+  External Metrics
+  ================
+    ConfigMap name: datadog-hpa
+    Number of external metrics detected: 2
 ```
-apiVersion: v1
-data:
-  external.metrics.default.nginxext-nginx.net.request_per_s: '{"name":"nginx.net.request_per_s","labels":{"kube_container_name":"nginx"},"ts":1530625676,"hpa_name":"nginxext","hpa_namespace":"default","value":0,"valid":false}'
-kind: ConfigMap
-metadata:
-  creationTimestamp: 2018-07-03T13:40:50Z
-  name: datadog-hpa
-  namespace: default
-  resourceVersion: "1742"
+If there is an error you will see it here.
+If you want a more verbose output, you can run the flare command:
```

Comment:
r/`If you want a more verbose output, you can run the flare command:`/`If you want a more verbose output, run the flare command:`